project/src/js/ui/layout.js

```diff
@@ -67,7 +67,7 @@ export default {
         {challengeForm.view()}
         {friendsPopup.view()}
         {lobby.view()}
-        {overlay ? overlay() : null}
+        {overlay ? overlay.map((t) => t()) : null}
```

Comment:
that is not necessary and is likely to break other views. 

Overlay should return an array of vdom, in the end it's the same as mapping over an array of functions returning a single vdom. But I'd like to not change layout api in fear to not break things